### DISCUSSION TITLE
automated builds for macOS

### DISCUSSION THREAD

**Discussion by Be-ing** [tenacityteam/tenacity-legacy#210] [General]
How can we make automated builds for macOS that does not require GitHub Actions? I am unsure if cross compiling from Linux to macOS is possible.

**Comment by falkTX** [comment]
I have gcc-4.2 working fine, though as that is very old now it is not much usable.

there are projects to run macOS insider qemu inside docker, for example https://github.com/Cleafy/sxkdvm

**Reply by Be-ing** [reply]
Interesting... that looks like it may be more practical than cross compiling.
